tests/test_rigid_benchmarks.py

```diff
@@ -25,8 +25,8 @@
 REPORT_FILE = "speed_test.txt"
 
 STEP_DT = 0.01
-NUM_WARMUP_FRAMES = 9500
-NUM_RECORD_FRAMES = 500
+DURATION_WARMUP = 45.0
+DURATION_RECORD = 15.0
```

Comment:
Will this value be stable on difference device / workload?

Reply:
For profiling it is the same, I record between 1min and 5min of data whatever the duration of the main loop.